submit/templates/submit/withdraw.html

```diff
@@ -0,0 +1,79 @@
+{%- extends "base/base.html" %}
+
+{% import "submit/submit_macros.html" as submit_macros %}
+
+{% block addl_head %}
+ <link rel="stylesheet" href="{{ url_for('static', filename='css/submit.css')}}" />
+ <script src="{{ url_for('static', filename='js/filewidget.js') }}"></script>
+{% endblock addl_head %}
+
+{% block content %}
+  <h1 class="title">Update journal reference</h1>
+  <p>
+    Articles that have been announced and made public cannot be completely
+    removed. However, you may submit a withdrawal notification for your
+    article.
+  </p>
+  {% if require_confirmation and not confirmed %}
+  <div class="notification is-info">
+    Please review the preview of your paper's abstract page, and confirm that
+    you with to proceed. You can continue to make changes until you are
```

Comment:
Please carefully check this preview, showing the abstract page as it will be updated. You can continue to make changes...